bands/site_sections/band_admin.py

```diff
@@ -49,40 +52,40 @@ def add_band(self, session, message='', **params):
         }
 
     @ajax
-    def mark_as_band(self, session, group_id):
+    def mark_as_band(self, session, group_id, group_type):
         group = session.group(group_id)
         if not group.leader:
             return {'message': '{} does not have an assigned group leader'.format(group.name)}
 
         if not group.band:
             group.band = Band()
+            group.band.group_type = group_type
             session.commit()
 
         return {
             'id': group.band.id,
-            'message': '{} has been marked as a band'.format(group.name)
+            'message': '{} has been marked as a {}'.format(group.name, group.band.group_type_label)
         }
 
     @ajax
     def remove_as_band(self, session, group_id):
         group = session.group(group_id)
+        group_type_label = group.band.group_type_label
 
         if group.band:
             group.band = None
             session.commit()
 
         return {
             'id': group.id,
-            'message': '{} has been removed as a band'.format(group.name)
+            'message': '{} has been removed as a {}'.format(group.name, group_type_label)
         }
 
-    def band_info(self, session, message='', event_id=None, **params):
+    def group_info(self, session, message='', event_id=None, **params):
         band = session.band(params)
         if cherrypy.request.method == 'POST':
             if event_id:
                 band.event_id = event_id
-            message = self._required_message(
```

Comment:
This is one of the stickier/less-well-solved parts of this refactor... there's columns on the Band object that are band-specific. However, since they all have default values, I figured it was okay to simply not require them from the admin.